config/mainCfg_ttCR_Legacy2016.cfg

```diff
@@ -0,0 +1,84 @@
+[general]
+
+lumi = 35922 # pb^-1 
+# the lumi in fb^-1 is used only by the plotting program
+# approximate to only one decimal digit
+lumi_fb = 35.9 # fb^-1
+
+outputFolder = /home/llr/cms/motta/CMSSW_11_1_0_pre6/src/KLUBAnalysis/MY_ttCRlimits_2016ALL/plots
+
+data = DTauB, DTauC, DTauD, DTauE, DTauF, DTauG, DTauH, DsingleMuB, DsingleMuC, DsingleMuD, DsingleMuE, DsingleMuF, DsingleMuG, DsingleMuH, DsingleEleB, DsingleEleC, DsingleEleD, DsingleEleE, DsingleEleF, DsingleEleG, DsingleEleH
+
+signals = GGHH_NLO_cHHH1_xs
+
+backgrounds = TTfullyHad, TTfullyLep, TTsemiLep, DY_0b_1Pt, DY_0b_2Pt, DY_0b_3Pt, DY_0b_4Pt, DY_0b_5Pt, DY_0b_6Pt, DY_1b_1Pt, DY_1b_2Pt, DY_1b_3Pt, DY_1b_4Pt, DY_1b_5Pt, DY_1b_6Pt, DY_2b_1Pt, DY_2b_2Pt, DY_2b_3Pt, DY_2b_4Pt, DY_2b_5Pt, DY_2b_6Pt, DY_LM, WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf, TWtop, TWantitop, singleTop_top, singleTop_antitop, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, WWTo2L2Nu, WWTo4Q, WWToLNuQQ, WZTo1L1Nu2Q, WZTo1L3Nu, WZTo2L2Q, WZTo3LNu, ZZTo2L2Nu, ZZTo2L2Q, ZZTo2Q2Nu, ZZTo4L, ZZTo4Q, ZH_HBB_ZLL, ZH_HBB_ZQQ, ZH_HTauTau, ggHTauTau, VBFHTauTau, WplusHTauTau, WminusHTauTau, ttHJetTononBB, ttHJetToBB, WWW, WWZ, WZZ, ZZZ, TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu, TTWW, TTWZ, TTZZ
+
+#variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, bjet1_pt, bjet2_pt, ditau_deltaR, tauH_SVFIT_mass, tauH_pt, bH_pt, bH_mass, HH_pt, HH_mass, DNNoutSM_kl_1
+variables = dau1_pt, bH_mass, DNNoutSM_kl_1
+
+selections = ttCR_invMcut, baseline
+
+regions    = SR, SStight, OSinviso, SSinviso
+
+[configs]
+sampleCfg = config/sampleCfg_Legacy2016.cfg
+cutCfg    = config/selectionCfg_ttCR_Legacy2016.cfg
+# pattern   = goodsystfiles 
+
+[merge]
+# For limits
+TT        = TTfullyHad, TTfullyLep, TTsemiLep
+W         = WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
+EWK       = EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL
+singleT   = singleTop_top, singleTop_antitop
+TW        = TWtop, TWantitop
+ZH        = ZH_HBB_ZLL, ZH_HBB_ZQQ, ZH_HTauTau
+WH        = WplusHTauTau, WminusHTauTau
+VV        = WWTo2L2Nu, WWTo4Q, WWToLNuQQ, WZTo1L1Nu2Q, WZTo1L3Nu, WZTo2L2Q, WZTo3LNu, ZZTo2L2Nu, ZZTo2L2Q, ZZTo2Q2Nu, ZZTo4L, ZZTo4Q
+ttH       = ttHJetTononBB, ttHJetToBB
+TTX       = TTZZ, TTWW, TTWZ, TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu
+ggH       = ggHTauTau
+VBFH      = VBFHTauTau
+VVV       = WWW, WWZ, WZZ, ZZZ
+DY_HM     = DY_0b_1Pt, DY_0b_2Pt, DY_0b_3Pt, DY_0b_4Pt, DY_0b_5Pt, DY_0b_6Pt, DY_1b_1Pt, DY_1b_2Pt, DY_1b_3Pt, DY_1b_4Pt, DY_1b_5Pt, DY_1b_6Pt, DY_2b_1Pt, DY_2b_2Pt, DY_2b_3Pt, DY_2b_4Pt, DY_2b_5Pt, DY_2b_6Pt
```

Comment:
The DY split in bins should not be merged, they should be left separate

Reply:
yeah this is only an intermediate step for the plotting, then it will remain separated. It is already separated in what I used for the datacards

Reply:
fixed